app/services/clients/connector.client.js

```diff
@@ -103,12 +103,8 @@ ConnectorClient.prototype = {
    */
   getAccount: function (params) {
     const url = _accountUrlFor(params.gatewayAccountId)
-    return baseClient.get({
-      baseUrl: this.connectorUrl,
-      url,
-      description: 'get an account',
-      service: SERVICE_NAME
-    })
+    return this.client._axios.get(url, { description: 'get an account' })
```

Comment:
is it possible to make it loo like `this.client.get(url)` for simplicity?

Reply:
Good idea, probably makes sense to do that - could also make `description` a parameter to a `get` method too as we'll always want to set it

Reply:
Have done this https://github.com/alphagov/pay-selfservice/pull/4142/commits/8788f01e16eed3e5af489602f7c209144c7e8711